charts/jindocache/templates/fuse/daemonset.yaml

```diff
@@ -158,20 +158,7 @@ spec:
           {{- end }}
           {{- end }}
           {{- if .Values.secret }}
-          {{- if .Values.UseStsToken }}
```

Comment:
Please also update chart version and change log.

Reply:
Done.